docs/en/docs/data-operate/import/import-way/broker-load-manual.md

```diff
@@ -43,38 +43,38 @@ After the user submits the import task, FE will generate the corresponding Plan
 
 BE pulls data from the broker during execution, and imports the data into the system after transforming the data. All BEs are imported, and FE ultimately decides whether the import is successful.
 
-````
+```
                  +
                  | 1. user create broker load
                  v
             +----+----+
-            | |
-            | FE |
-            | |
+            |         |
```

Comment:
the Chinese version may has the same problem, should fix them together.

Reply:
Have check the following links and the Chinese version is good. 
- https://github.com/apache/doris/blob/master/docs/zh-CN/docs/data-operate/import/import-way/broker-load-manual.md#%E5%9F%BA%E6%9C%AC%E5%8E%9F%E7%90%86
- https://doris.apache.org/zh-CN/docs/dev/data-operate/import/import-way/broker-load-manual/